internal/web3ext/web3ext.go

```diff
@@ -792,6 +792,10 @@ web3._extend({
 			call: 'istanbul_removeProxy',
 			params: 1
 		}),
+		new web3._extend.Property({
```

Comment:
not sure who this is done, but as we have `web.eth` `web.ssh` and other... maybe it would be nice to have `web.ibft` here

Reply:
Or some other namespace. Not istanbul, since that's also the name for eth2.0

Reply:
Agreed that we should rename the `istanbul` rpc namespace to something different.   In fact, I think we should rename the geth istanbul package to that new name (btw, we really should come up with a name for our POS/Consensus mechanism.  All other blockchain companies did that).

IMO, that would be a much larger ticket.

Reply:
just the package rename is happening just after 1.9 or even there... since it already has "istanbul" as a thing

Reply:
Oh really?  That's interesting.  So sounds like IBFT will be a "first class" engine.

Reply:
Actually, I think I may have misread your comment.  I thought that you meant that geth 1.9 already has the istanbul package.   But after re-reading, I think when you say `1.9`, you mean to refer to the task of merging 1.9 into our implementation.